### PR TITLE
Etfe 2171 - DES07 destination Warehouse Vat

### PR DESCRIPTION
**Pull request by funkmaster20th**

### CODE REVIEW

**Comment by grahampaulcook** [comment]
test this please

**Comment by funkmaster20th** [comment]
the guarantorRequired in themessages file is duplicated? and also the destination.subheading.

**Comment by funkmaster20th** [comment]
Screenshot 2023-10-17 at 13 55 32
Screenshot 2023-10-17 at 13 55 43
Screenshot 2023-10-17 at 13 55 55
Screenshot 2023-10-17 at 13 56 09

**Comment by funkmaster20th** [comment]
Screenshot 2023-10-17 at 14 22 22

File: app/pages/sections/destination/DestinationWarehouseVatPage.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object DestinationWarehouseVatPage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
```
Comment (grahampaulcook) [inline app/pages/sections/destination/DestinationWarehouseVatPage.scala:23]:
```suggestion
  override def path: JsPath = JsPath \ "destination" \ toString
```

File: app/views/sections/destination/DestinationWarehouseVatView.scala.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            )
+            .withWidth(Full)
+        )
+
+    @continueOrSaveOrSkip(skipQuestionCall.url)
+
+
+    }
+}
+
```
Comment (grahampaulcook) [inline app/views/sections/destination/DestinationWarehouseVatView.scala.html:60]:
```suggestion
@{
        //$COVERAGE-OFF$
    }
```

**Pull request merged**